nevergrad/benchmark/experiments.py

```diff
@@ -30,6 +30,63 @@
 # fmt: off
 
 
+@registry.register
+def yawidebbob(seed: Optional[int] = None) -> Iterator[Experiment]:
+    seedg = create_seed_generator(seed)
+    # Continuous case
+    # First, a few functions with constraints.
+    functions = [
+        ArtificialFunction(name, block_dimension=50, rotation=rotation) for name in ["cigar", "ellipsoid"] for rotation in [True, False]
+    ]
+    for func in functions:
+        func.parametrization.register_cheap_constraint(_positive_sum)
+    # Then, let us build a constraint-free case. We include the noisy case.
+    names = ["hm", "rastrigin"]  #, "griewank", "rosenbrock", "ackley", "lunacek", "deceptivemultimodal", "bucherastrigin", "multipeak"]
+    names += ["sphere", "doublelinearslope", "stepdoublelinearslope"]
+    names += ["cigar", "ellipsoid", "stepellipsoid"]  #"altcigar", "altellipsoid", "stepellipsoid", "discus", "bentcigar"]
+    # names += ["deceptiveillcond", "deceptivemultimodal", "deceptivepath"]
+    functions = [
```

Comment:
all functions definitions before this are lost, did you mean `+=`?

Reply:
thx!